test/idp/src/main/kotlin/org/codice/compliance/web/sso/PostLoginTest.kt

```diff
@@ -46,9 +46,12 @@ class PostLoginTest : StringSpec() {
                     .`when`()
                     .post(Common.getSingleSignOnLocation(SamlProtocol.POST_BINDING))
 
-            response.statusCode shouldBe 200
             val idpResponse = getServiceProvider(IdpResponder::class.java).getIdpPostResponse(response)
-            verifyResponse(idpResponse)
+
+            val idpResponseDecorator = IdpResponseDecoratorFactory.getDecorator(idpResponse)
+
+            val bindingVerifier = BindingVerifierFactory.getBindingVerifier(idpResponseDecorator)
+            bindingVerifier.verify()
```

Comment:
Make sure you rebase because I made some changes in master on the verifications that are happening here. But my comment still holds for the ones you're adding.